modules/core/04-channel/keeper/upgrade.go

```diff
@@ -36,101 +27,32 @@ func (k Keeper) ChanUpgradeInit(
 		return 0, "", errorsmod.Wrapf(types.ErrInvalidChannelState, "expected %s, got %s", types.OPEN, channel.State)
 	}
 
-	if !k.scopedKeeper.AuthenticateCapability(ctx, chanCap, host.ChannelCapabilityPath(portID, channelID)) {
-		return 0, "", errorsmod.Wrapf(types.ErrChannelCapabilityNotFound, "caller does not own capability for channel, port ID (%s) channel ID (%s)", portID, channelID)
-	}
```

Comment:
I know its kind of pseudo auth, but this was consistent with the opening handshake handlers. Are we good with removing this capability authentication entirely?

Reply:
context: https://github.com/cosmos/ibc/issues/886